code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -271,7 +271,7 @@
 									"<span class='notice'>You cut \the [src] apart with \the [WT].</span>",
 									"<span class='italics'>You hear welding.</span>")
 					var/turf/T = get_turf(src)
-					new material_drop(T)
+					new material_drop(T, 2)

```

Comment:
This isn't very future proof.

This should be a var on closets called `material_drop_amount` with it defaulting to 2---invariably, we're going to get some closet that needs more (or less) than 2 sheets when it's deconstructed.